### DISCUSSION TITLE
User for spin provision and maintain

### DISCUSSION THREAD

**Discussion by schnetzi** [serversideup/spin#147] [Ideas]
Would it make sense to use the first user in the `spin.yml` > `users` for `spin provision` and `spin maintain`? Currently I always need to manually pass the user.

**Comment by jaydrogers** [comment]
Are you able to have your user from your system (run `whoami`) match your username set in `.spin.yml`?

That might save you from specifying a user all the time.

Assuming the first user in the list could get messy since there is a potential the `.spin.yml` could be encrypted and shared with trusted contacts on a team.
